docs/kratos/concepts/credentials/username-email-password.mdx

```diff
@@ -410,6 +410,76 @@ $firescrypt$ln=<mem_cost>,r=<rounds>,p=<parallelization>$<salt>$<hash>$<salt_sep
 $firescrypt$ln=14,r=8,p=1$sPtDhWcd1MfdAw==$xbSou7FOl6mChCyzpCPIQ7tku7nsQMTFtyOZSXXd7tjBa4NtimOx7v42Gv2SfzPQu1oxM2/k4SsbOu73wlKe1A==$Bw==$YE0dO4bwD4JnJafh6lZZfkp1MtKzuKAXQcDCJNJNyeCHairWHKENOkbh3dzwaCdizzOspwr/FITUVlnOAwPKyw==
 ```
 
+### MD5
+
+Ory Identities doesn't hash password by MD5 but can compare stored MD5 hashes and migrate to configured hasher
+(`hashers.algorithm`). MD5 hashes can be configured in two different ways. Format is dependent on whether the hash is salted or
+not.
+
+#### Plain Format (for hashes without salt)
+
+```
+$md5$<hash>
+```
+
+#### Parameters
+
+- `hash`(`string`): The computed hash by the MD5 algorithm encoded in base64.
```

Comment:
I'm a little bit unsure about the "Encoded to Base64" sentence since it's used as "Encoded in base64" in many other places

For example:
https://github.com/ory/docs/blob/94ff2978b589c1fd4cf5c3dcd5b7a5da63d37e23/docs/kratos/concepts/credentials/username-email-password.mdx?plain=1#L327
https://github.com/ory/docs/blob/94ff2978b589c1fd4cf5c3dcd5b7a5da63d37e23/docs/kratos/concepts/credentials/username-email-password.mdx?plain=1#L352-L353
https://github.com/ory/docs/blob/94ff2978b589c1fd4cf5c3dcd5b7a5da63d37e23/docs/kratos/concepts/credentials/username-email-password.mdx?plain=1#L377-L378
https://github.com/ory/docs/blob/94ff2978b589c1fd4cf5c3dcd5b7a5da63d37e23/docs/kratos/concepts/credentials/username-email-password.mdx?plain=1#L402-L405

Reply:
That is true, good point. We're often updating docs in "waves" so there might be things that are not yet changed to the way we want them to be. This looks like one of them. 

There are two considerations here: 
1) capitalization of "Base64" - I used a capital "B" based on the way it's done in multiple 1st-search-page sources
2) The preposition "to" - I used "to" instead of "in" as you take a non-encoded input and then run it through a process that turns it TO a format that is Base64 - that's my train of thought here. Does it sound right? I'm not an engineer so I'm trying to figure it out looking from a language pov supported by reading up on the tech a bit.

Reply:
Well actually you are most certainly right about how the process goes. It's just that, in general when developers mentions on some string's encoding, they use `in` as preposition instead of `to`.  But I've got to say that I've seen usages of `into` when some documentary mentions about "changing the encoding of the string". (For example: https://en.wikipedia.org/wiki/Base64)

To be honest though,  `to`'s usage in here doesn't seem odd or wrong to me at all. So, if you wanted me to change it to `to` (`to to` that's a bit weird sentence isn't it ? :smile: ), I would totally get it. I'm going to make the change according to your following decision on this.